renderer/components/Home/WalletSettingsFormLocal.js

```diff
@@ -48,6 +61,32 @@ class WalletSettingsFormLocal extends React.Component {
 
           <DataRow left={<FormattedMessage {...messages.chain} />} py={2} right={chain} />
           <DataRow left={<FormattedMessage {...messages.network} />} py={2} right={network} />
+
+          <Bar variant="light" />
+          <DataRow
+            left={<FieldLabel itemKey="neutrinoUrl" tooltip="neutrinoUrl_tooltip" />}
+            right={
+              <TextArea
```

Comment:
Can we add some kind of validation to this textarea to ensure that what is entered is a contactable address? Perhaps use `dns.lookup` like we do when validation remove node addresses?

Otherwise, I can enter something like `w` on a newline and then the wallet fails to start with an error like `Unable to initialize neutrino backend: unable to create neutrino light client: lookup w: no such host`

Reply:
Also, I can do something like:
```
mainnet1-btcd.zaphq.io
mainnet2-btcd.zaphq.io
mainnet2-btcd.zaphq.io
```

And end up with a startup command like `--neutrino.connect=mainnet1-btcd.zaphq.io --neutrino.connect=mainnet2-btcd.zaphq.io --neutrino.connect=mainnet2-btcd.zaphq.io`. Whilst this is valid, I'm not sure that it's desirable. Might be an idea to validate against that sort of things

Reply:
Validation is quite tricky because it is async. But believe i made a lot of progress towards this. Check the updated PR